db/migrate/20180412153251_add_split_feature_gate.rb

```diff
@@ -0,0 +1,6 @@
+class AddSplitFeatureGate < ActiveRecord::Migration[5.0]
+  def change
+    add_column :splits, :feature_gate, :boolean, default: false, null: false
```

Comment:
This is a high throughput table but very small number of rows and there are effectively no long-running transactions on this service. My hope is this doesn't cause issues on deploy. :) /cc @Betterment/data for commentary.

Reply:
👍 not sure what our tt deploy pattern is, maybe we want to deploy to stage first and watch the migration?

Reply:
Won’t do much for lock contention confidence because of the vastly different load levels but yeah. Will do this and the other off hours unless data says no.